src/views/Eboard.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div>
+    <section>
+      <div class="parallax-one">
+        <h1>
+          Executive
+          <br />Board
+        </h1>
+      </div>
+    </section>
+    <div class="ePeeps">
+      <EboardPeople v-bind:eboard="eboard" />
+    </div>
+  </div>
+</template>
+
+<script>
+import EboardPeople from "../components/EboardPeople";
+
+export default {
+  name: "eboard",
+  components: {
+    EboardPeople
+  },
+  data() {
+    return {
+      eboard: [
+        {
```

Comment:
Add this data to the Vuex store. 
Use this PR as a reference: https://github.com/Code-4-Community/c4cneu-frontend/pull/3